app/src/main/java/br/com/compass/compassmart/ui/onboarding/OnboardingFragment.kt

```diff
@@ -0,0 +1,86 @@
+package br.com.compass.compassmart.ui.onboarding
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.widget.ViewPager2
+import br.com.compass.compassmart.R
+import br.com.compass.compassmart.databinding.FragmentOnboardingBinding
+import br.com.compass.compassmart.ui.onboarding.model.InformacoesModel
+
+class OnboardingFragment : Fragment() {
+
+    private val binding: FragmentOnboardingBinding by lazy {
+        FragmentOnboardingBinding.inflate(layoutInflater)
+    }
+    private var data = mutableListOf<InformacoesModel>()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
```

Comment:
Precisa adicionar essa linha agora, por conta da nova implementação do binding.

```suggestion
_binding = ResultProfileBinding.inflate(inflater, container, false)
```

Reply:
Adicionada.